src/uint/inv_mod.rs

```diff
@@ -25,11 +27,72 @@ impl<const LIMBS: usize> UInt<LIMBS> {
         }
         x
     }
+
+    /// Computes the multiplicative inverse of `self` mod `modulus`. In other words `self^-1 mod modulus`. Returns `(inverse, 1...1)` if an inverse exists, otherwise `(undefined, 0...0)`. The algorithm is the same as in GMP 6.2.1's `mpn_sec_invert`.
+    pub const fn ct_inv_mod(self, modulus: &UInt<LIMBS>) -> (Self, Word) {
+        debug_assert!(modulus.ct_is_odd() == Word::MAX);
+
+        let mut a = self;
+
+        let mut u = UInt::ONE;
+        let mut v = UInt::ZERO;
+
+        let mut b = *modulus;
```

Comment:
If you're derefing anyway, perhaps consider making `modulus` owned instead of borrowed?